gcloud-java-dns/src/main/java/com/google/gcloud/dns/Dns.java

```diff
@@ -182,7 +158,7 @@ public String selector() {
   /**
    * Class for specifying record set listing options.
    */
-  class RecordSetListOption extends AbstractOption implements Serializable {
+  class RecordSetListOption extends Option implements Serializable {
```

Comment:
Unrelated to the PR but while you are at it, do you mind to remove the redundant `Serializable` for classes
that extends `Option` in this file?